eoscostarica/eos-rate
# Refactor/rmrate

### GH Issue

Change if conditional to checker check to report the user that cannot remote a rate if it doesn't exist 

Resolves #747 

### Steps to test

1. Remote a rate
1. Call `rmrate` action from terminal or webapp
1. Check a error message is returned saying "Rate should exist before to try to remove it"
